packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts

```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
+      },
     );
-  });
 
-  test("invoke Web3SigningCredentialType.GETHKEYCHAINPASSWORD", async (t2: Test) => {
-    const newName = `DrCactus${uuidV4()}`;
-    const setNameOut = await connector.getContractInfoKeychain({
-      contractName,
-      keychainId: keychainPlugin.getKeychainId(),
-      invocationType: EthContractInvocationType.Send,
-      methodName: "setName",
-      params: [newName],
+    await connector.transact({
       web3SigningCredential: {
         ethAccount: firstHighNetWorthAccount,
         secret: "",
         type: Web3SigningCredentialType.GethKeychainPassword,
       },
-      nonce: 2,
+      transactionConfig: {
+        from: firstHighNetWorthAccount,
+        to: testEthAccount.address,
+        value: 10e9,
+      },
     });
-    t2.ok(setNameOut, "setName() invocation #1 output is truthy OK");
 
-    try {
-      const setNameOutInvalid = await connector.getContractInfoKeychain({
+    const balance = await web3.eth.getBalance(testEthAccount.address);
+    expect(balance).toBeTruthy();
+    expect(parseInt(balance, 10)).toEqual(10e9);
+    // t.equals(parseInt(balance, 10), 10e9, "Balance of test account is OK");
+
+    let contractAddress: string;
+
+    {
+      // test("deploys contract via .json file", async (t2: Test) => {
+      const deployOut = await connector.deployContract({
+        keychainId: keychainPlugin.getKeychainId(),
+        contractName: HelloWorldContractJson.contractName,
+        web3SigningCredential: {
+          ethAccount: firstHighNetWorthAccount,
+          secret: "",
+          type: Web3SigningCredentialType.GethKeychainPassword,
+        },
+        gas: 1000000,
+      });
+      expect(deployOut).toBeTruthy();
+      expect(deployOut.transactionReceipt).toBeTruthy();
+      expect(deployOut.transactionReceipt.contractAddress).toBeTruthy();
+
+      contractAddress = deployOut.transactionReceipt.contractAddress as string;
+      expect(typeof contractAddress === "string").toBeTrue();
+      // t2.ok(
+      //   typeof contractAddress === "string",
+      //   "contractAddress typeof string OK",
+      // );
+
+      const { callOutput: helloMsg } = await connector.getContractInfoKeychain({
+        contractName,
+        keychainId: keychainPlugin.getKeychainId(),
+        invocationType: EthContractInvocationType.Call,
+        methodName: "sayHello",
+        params: [],
+        web3SigningCredential: {
+          ethAccount: firstHighNetWorthAccount,
+          secret: "",
+          type: Web3SigningCredentialType.GethKeychainPassword,
+        },
+      });
+      expect(helloMsg).toBeTruthy();
+      // t2.ok(helloMsg, "sayHello() output is truthy");
+      expect(typeof helloMsg === "string").toBeTrue();
+      // t2.true(
+      //   typeof helloMsg === "string",
+      //   "sayHello() output is type of string",
+      // );
+      // });
+    }
+
+    // test("invoke Web3SigningCredentialType.GETHKEYCHAINPASSWORD", async (t2: Test) => {
```

Comment:
@Leeyoungone No commented code unless you need it for something specific please :-)